### PR TITLE
doc: improve the removal of monitor

### PR DESCRIPTION
**Pull request by kifeo**
In the changing a monitor's IP address (the Right way) part, it is told to remove the entry, where it is told to remove the ceph.conf entry.
However, it is never told to add the entry.
This is the purpose of this change.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by neha-ojha** [comment]
Could you please add a `Signed-off-by` line in your commit messages? More details in https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#sign-your-work

File: doc/rados/operations/add-or-rm-mons.rst [hunk 1]
```diff
@@ -133,6 +133,14 @@ on ``mon.a``).
    For example::
 
 	ceph-mon -i {mon-id} --public-addr {ip:port}
+	
+
+#. add the entry in the ceph.conf configuration file.
```
Comment (tchaikov) [inline doc/rados/operations/add-or-rm-mons.rst:138]: might want to note that 

1. this step is *optional*. as the clients are able to discover and fallback to the new monitor if it is able to connect to the cluster. but it helps to share the load from a newly started a client.
2. please replace, for instance, `d` in `[mon.d]` with `[mon{mon-id}]` or note that these settings are subject to change according to the actual settings of the newly added monitor.

Comment (tchaikov) [reply doc/rados/operations/add-or-rm-mons.rst:138]: and the title could be improved, as the added step belongs to the section of "Adding a Monitor (Manual)".

**Comment by ljflores** [comment]
@kifeo any updates?

**Comment by kifeo** [comment]
@ljflores, seems my PR is not needed as per @tchaikov comment, however, I had to perform the step manually to add the monitor in the file from my side.

File: doc/rados/operations/add-or-rm-mons.rst [hunk 2]
```diff
@@ -133,6 +133,14 @@ on ``mon.a``).
...
+   For example::
+   
+   	[mon.d]
+		host = host04
+		addr = 10.0.0.4
```
Comment (ljflores) [inline doc/rados/operations/add-or-rm-mons.rst:143]:
```
[mon.{mon-id}]
host = {hostname}
addr = {ip-address}
```

File: doc/rados/operations/add-or-rm-mons.rst (hunk 1 above)
Comment (ljflores) [reply doc/rados/operations/add-or-rm-mons.rst:138]:
```
_Optional -- only complete this step if {insert reason}:_ Add the entry in the ceph.conf configuration file. For example:
```

**Pull request closed without merging**